RELEASE_NOTES.md

```diff
@@ -12,6 +12,9 @@
 - Localization issues in address lookup functionality.
 - Overriding some of the XML styles without specifying a parent style no longer causes a build error.
 
+## Deprecated
```

Comment:
I think this should be removed instead of deprecated?

Reply:
Fair enough, forgot we had that tag